assets/js/src/core/modules/asset/editor/shared-tab-manager/tabs/notes-and-events/notes-and-events-view.style.tsx

```diff
@@ -22,12 +22,50 @@ export const useStyles = createStyles(({ token, css }) => {
 
   return {
     'notes-and-events': css`
-      display: flex;
-      flex-direction: row;
-      height: 100%;
-      width: 100%;
-      overflow: hidden;
- 
+
+      span, div, div.anticon, button {
+        vertical-align: middle;
+      }
+
+      .panel-title {
+        font-size: 12px;
+        font-weight: 600;
+      }
+
+      .panel-title__user, .panel-title__divider {
+        vertical-align: unset;
+        font-weight: 400;
+        line-height: 20px;
+        color: ${token.colorTextSecondary}
+      }
+
+      .panel-extra__trash-icon {
+        color: ${token.colorPrimary};
+      }
+
+      .panel-body__description {
+        display: block;
+        margin: 0;
+      }
+
+      .panel-body__description-padding {
+        padding-bottom: ${token.paddingSM}px;
+      }
+
+      .panel-body__details {
+        display: block;
+        padding-bottom: ${token.paddingXS}px;
+      }
+
+      .ant-table-wrapper p {
+        margin: 0;
+      }
+
+      .ant-table-wrapper .ant-table-cell {
+        white-space: unset;
+        overflow-wrap: anywhere;
+      }
```

Comment:
We have a lot of new styles here. Why do we need them?

Reply:
@vin0401 They have been moved from the notes-and-events-card.style and refer to the part of the code where we are able to display a grid if there is note.data defined. 
I have not been sure how to create that scenario. Would you know how?

![Screenshot 2024-09-11 at 12 28 38](https://github.com/user-attachments/assets/0f0d294d-bdad-4f18-a934-78344a37d667)

Reply:
okay, I think we can leave it like that for now. And hopefully we can remove it, the moment we have some of the spacing components.